zoo-css-challenge/index.html

```diff
@@ -21,6 +21,14 @@
  	<main>
  		<section class="introduction">
  			<h1>Birmingham Zoo</h1>
+			<p> The zoo is open every day of the year and features three major biomes: 
+				the Tropic Zone, Temperate Territory, and the Polar Circle. From tropical birds, to snow leopards, 
+				grizzly bears, and one of the nation’s largest colonies of Antarctic penguins, 
+				there are animals to enjoy in every season.</p>
+			<p> 	The zoo is open every day of the year and features three major biomes: the Tropic Zone, 
+				Temperate Territory, and the Polar Circle. From tropical birds, to snow leopards, 
+				grizzly bears, and one of the nation’s largest colonies of Antarctic penguins, 
+				there are animals to enjoy in every season.</p>
```

Comment:
```suggestion
```
It looks like we've added the paragraph twice — we can delete the duplicate paragraph